src/main/scala/thurloe/notification/NotificationMonitor.scala

```diff
@@ -194,12 +194,29 @@ class NotificationMonitorActor(val pollInterval: FiniteDuration,
 
   def lookupShouldNotify(notification: Notification): Future[Boolean] =
     notification match {
+      //For workspace notifications, there are three tiers of preferences to check:
+      // 1. has the user disabled *all* notifications for their account?
+      //     key format: notifications.off
+      // 2. has the user disabled the notification at the type-level?
+      //     key format: notifications/<notification class name>
+      // 3. has the user disabled the notification for the specified workspace?
+      //     key format: notifications/<notification class name>/<workspace namespace>/<workspace name>
+      case wsNotification: WorkspaceNotification =>
+        val baseKey = s"notifications/${wsNotification.getClass.getSimpleName}"
+        //Check #1
```

Comment:
If this check is to be labeled in a comment, should checks 2 and 3 also be labeled?

Reply:
Comment was leftover from me mapping out how this logic should work, I'll just remove this comment